fieldpath/set_test.go

```diff
@@ -192,41 +192,44 @@ func TestSetIntersectionDifference(t *testing.T) {
 		MakePathOrDie("bar", "c2"),
 	)
 
-	i := NewSet(
-		MakePathOrDie("a1"),
-		MakePathOrDie("foo", 1),
-		MakePathOrDie("b1", KeyByFields("name", value.StringValue("first"))),
-	)
-
-	got := s1.Intersection(s2)
-
-	if !got.Equals(i) {
-		t.Errorf("s1 intersect s2: expected: \n%v\n, got \n%v\n", i, got)
-	}
-
-	sDiffS2 := NewSet(
-		MakePathOrDie("a0"),
-		MakePathOrDie("foo", 0),
-		MakePathOrDie("b0", KeyByFields("name", value.StringValue("first"))),
-		MakePathOrDie("bar", "c0"),
-	)
-
-	got = s1.Difference(s2)
-
-	if !got.Equals(sDiffS2) {
-		t.Errorf("s1 - s2: expected: \n%v\n, got \n%v\n", sDiffS2, got)
-	}
-
-	s2DiffS := NewSet(
-		MakePathOrDie("a2"),
-		MakePathOrDie("foo", 2),
-		MakePathOrDie("b2", KeyByFields("name", value.StringValue("first"))),
-		MakePathOrDie("bar", "c2"),
-	)
-
-	got = s2.Difference(s1)
-
-	if !got.Equals(s2DiffS) {
-		t.Errorf("s2 - s1: expected: \n%v\n, got \n%v\n", s2DiffS, got)
-	}
+	t.Run("intersection", func(t *testing.T) {
```

Comment:
Quite minor, but I'm not really convinced that this is a good use-case for `Run`. In general I'd keep tests as small and isolated as possible. Whatever.

Reply:
Hopefully we can agree that this is better than importing ginkgo ;)

Reply:
:joy: Absolutely!